server/lib/enrich/article.js

```diff
@@ -33,6 +33,14 @@ module.exports = exports = function article(content, format) {
 
 	content.hasGraphics = Boolean(content.contentStats && content.contentStats.graphics);
 
+	// Currently embeds can have more than one item for each picture - for different screen sizes
+	// We are assuming that canBeSyndicated is the same for all sizes of a picture
+	// We are asking if ALL Graphics can be syndicated, which means as long as at least one item
+	// can't be, the answer to this is 'no'
+	const atLeastOneGraphicCantBeShared = content.embeds && content.embeds.filter(embed => embed && embed.type.endsWith('Graphic')).some(item => item.canBeSyndicated !== 'yes');
+
+	content.canAllGraphicsBeSyndicated = Boolean(atLeastOneGraphicCantBeShared);
```

Comment:
This is confusing for me - the content property is a positive assertion but then its being converted to a boolean of a negative assertion?

Reply:
Well spotted - confused myself with the double negatives and all 😅